src/go/k8s/helm-chart/charts/redpanda-operator/README.md

```diff
@@ -0,0 +1,53 @@
+# Redpanda Operator
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+## Installation
+
+### Prerequisite
+
+Due to problems with helm updating the CRD and deletion can destroy all custom resource, the CRD
+installation is moved to different step.
+
+To install Redpanda operator CRDs please run the following command:
+
+```sh
+kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd
```

Comment:
I think ideally we should point to a tag here, not dev. This will be problematic when we work on dev and introduce new feature. We don't have a tag right now though :( Open an issue please to do this after the next release?

Reply:
Agree, I will change it to the latest tag (`v21.3.4`).